test/tests/concrete_ASR_swelling/tests

```diff
@@ -14,7 +14,19 @@
       custom_cmp = 'asr_confined.cmp'
       detail = 'with anisotropic expansion strains, '
     []
-
+    [test_strip_kelvin]
+      type = 'Exodiff'
+      input = 'asr_confined.i'
+      exodiff = 'asr_confined_strip_kelvin_out.e'
```

Comment:
Can you actually exodiff against the base case? You can do that by making a symbolic link from the _kelvin version to the base version of the file in the gold directory.

Reply:
If I make exodiff against the base class, it show diff in temperature as expected. As of now I do not know how to do symbolic link, but I will try and let you know.

Reply:
I have included the symbolic link file in gold directory and referred it for comparison.